react-hot-loader/index.d.ts

```diff
@@ -0,0 +1,20 @@
+// Type definitions for react-hot-loader
+// Project: https://github.com/gaearon/react-hot-loader
+// Definitions by: Jacek Jagiello <https://github.com/jacekjagiello/>
+// Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
+
+import * as React from "react"
+
+declare module "react-hot-loader" {
+
+  interface ErrorReporterProps {
+    error: any
+  }
+
+  export interface AppContainerProps {
+    children?: React.ReactElement<any>,
```

Comment:
You are right, but there is a problem with stateless components and `children` props, which probably will be resolved in typescript 2.1: https://github.com/Microsoft/TypeScript/issues/8588